packages/db/src/index.ts

```diff
@@ -0,0 +1 @@
+export * from '@prisma/client';
```

Comment:
For now, this package just exports the prisma client. Doing it this way will allow us to eventually bundle the client in this package so we could support multiple prisma schemas.